routes/article.js

```diff
@@ -110,9 +110,11 @@ router.get('/new', (req, res) => {
 router.get('/:id', (req, res) => {
     Article.findById(req.params.id)
         .then((article) => {
+            if (!article) throw new Error('No articles found')
             res.send(article) 
         })
         .catch((err) => {
+            err = err.toString()
             res.status(500).send({
                 err,
```

Comment:
It could be better practise, I just don't know the benefits of it yet. But it seemed like you were trying to get a `String` in the end anyways so I didn't see the use for the `Error` object